web/vite.config.ts

```diff
@@ -18,11 +18,20 @@ export default defineConfig({
   plugins: [
     // disable hot module reloading during test runs
     svelte({ hot: !process.env.VITEST }),
-    wasmPack(["../backend"], ["route-snapper"])
+    wasm(),
   ],
+  server: {
+    fs: {
+      allow: ["./", "../backend/pkg"],
```

Comment:
I dunno why importing from "backend" doesn't work, but it doesn't, and we need to tell vite it's OK to look out of the web/ dir